components/widgets/title/index.js

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components'
+
+const Title = styled.h1`
+  flex: 1 1 100%;
+  font-size: 60px;
```

Comment:
At the moment all sizes are based on `em` instead of `px`. Can you change that?

Reply:
Fixed.